modules/test/playwright/tests/portal-workflow-task-web/assignments.spec.ts

```diff
@@ -6,76 +6,158 @@
+	workflowReviewTasksPage,
+	workflowTasksPage,
+	page,
+	scriptManagementPage,
+	workflowPage
+}) => {	
+	workflowDefinitionName = 'Workflow Definition' + getRandomString();
 
-	workflowDefinitionName = 'Copy of Single Approver' + getRandomInt();
+	workflowXMLDefinition = readFileSync(__dirname + '/dependencies/test-workflow-definition.xml', 'utf-8');
 
 	const workflowDefinition =
 		await apiHelpers.headlessAdminWorkflow.postWorkflowDefinitionSave(
 			workflowDefinitionName,
-			singleApproverWorkflowDefinition
+			{ content: workflowXMLDefinition }
 		);
 
 	workflowDefinitionId = workflowDefinition.id;
-});
 
-test.afterEach(async ({apiHelpers}) => {
-	await apiHelpers.headlessAdminWorkflow.deleteWorkflowDefinition(
-		workflowDefinitionId
+	role = await apiHelpers.headlessAdminUser.postRole({
+        name: '5fd79328-7e81-4861-ae96-8b86e3d65524',
+    });
+
+	const user1 = await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+		"test@liferay.com"
 	);
-});
 
-test('por favor funcione', async ({
-	apiHelpers,
-    diagramViewPage,
-	nodePropertiesSidebarPage,
-	page,
-	processBuilderPage,
-	scriptManagementPage,
-	workflowPage
-}) => {
+	const user2 = await apiHelpers.headlessAdminUser.postUserAccount();
 
-	const role = await apiHelpers.headlessAdminUser.postRole({
-        name: 'Teacher' + getRandomString(),
-    });
+	await apiHelpers.headlessAdminUser.assignUserToRole('5fd79328-7e81-4861-ae96-8b86e3d65524', user1.id);
+	await apiHelpers.headlessAdminUser.assignUserToRole('5fd79328-7e81-4861-ae96-8b86e3d65524', user2.id);
 
-    const user = await apiHelpers.headlessAdminUser.postUserAccount();
+	await processBuilderPage.goto();
+
+	await processBuilderPage.clickWorkflowDefinitionName(
+		workflowDefinitionName
+	);
+
+	await diagramViewPage.publishWorkflowDefinition();
+
+	await diagramViewPage.goBack();
 
-    await apiHelpers.headlessAdminUser.assignUserToRole(role.name, user.id);
+	await configurationTabPage.goTo();
 
-	await workflowPage.goto();
+	assetType = 'Blogs Entry';
 
-	await workflowPage.changeWorkflow(
+	await configurationTabPage.assignWorkflowToAssetType(
 		workflowDefinitionName,
-		'Single Approver'
+		assetType
 	);
-	
-	await processBuilderPage.goto();
 
-	await processBuilderPage.clickWorkflowDefinitionName('Single Approver');
+	await blogsPage.goto();
+
+	await blogsPage.goToCreateBlogEntry();
+
+	blogTitle = 'Blog Title' + getRandomInt();
+
 
-	await diagramViewPage.clickNode('review');
+	await blogsEditBlogEntryPage.editBlogEntry({
+		content: 'Blog content.',
+		submitToWorkflow: true,
+		title: blogTitle,
+	});
+
+	await workflowTasksPage.goToAssignedToMyRoles();
+
+	await page.waitForLoadState('networkidle');
+
+	await workflowTasksPage.goToReviewPage(blogTitle);
+
+	await page.waitForTimeout(3000);
+
+	await workflowReviewTasksPage.clickReviewActionMenu();
+
+	await workflowReviewTasksPage.clickAssignToMenuItem();
+		
+	await workflowReviewTasksPage.selectAssignee(user2.id.toString());
```

Comment:
Hey Victor, I can se the interface declaration of user:

```
type TUserAccount = {
	alternateName?: string;
	emailAddress?: string;
	familyName?: string;
	givenName?: string;
	id?: string;
	name?: string;
};
```

I noticed that this ID is already a string. Could you check in the API return if this property is actually a string? If it is a number, I believe we can correct this interface.